packages/typescript/internal/ts_project.bzl

```diff
@@ -64,6 +75,19 @@ def _join(*elements):
         return "/".join(segments)
     return "."
 
+def _relative_to_package(path, ctx):
```

Comment:
nit: on the 5.x branch we can just `load("@bazel_skylib//lib:paths.bzl"` to get the relativize helper, but maybe not enough code sharing that it's worth it